src/css/flags/flags.css

```diff
@@ -5,7 +5,7 @@
 }
 .f-and,
-.f-andorra {
+.f-ad {
```

Comment:
hi @VizualAbstract here it is ok to have only `.f-and`. Could you please consider to remove `f-ad`?
The CSS class named "ad" creates issue with AdBlocks.